package-examples/helloworld-kustomize/README.md

```diff
@@ -0,0 +1,67 @@
+# helloworld-kustomize
+
+This is an example of a kpt package that has a kustomize
```

Comment:
It's better to summarize the purpose and end goal here.

Reply:
I think this sample doesn't make 100% sense TBH.  I am not sure we are going to publicize doing kpt setters that change kustomize packages.

Reply:
We will need to get these running but after that re-think what are the things that we want to showcase.